.gitignore

```diff
@@ -65,3 +65,4 @@ cypress
 
 # e2e test directory
 test/e2e
+.vscode
```

Comment:
+1 on `.vscode` but maybe we should straight use https://www.gitignore.io/api/vim%2Ccode%2Clinux%2Cmacos%2Cwindows%2Csublimetext%2Cnode?

Reply:
Somehow today I decided to use vs code. We can update that later

Reply:
Alright, created https://github.com/contentful/the-example-app.nodejs/issues/79